lib/telegram_mailer_patch.rb

```diff
@@ -106,8 +105,13 @@ def issue_add_with_telegram(issue, to_users, cc_users)
         :short => true
       } if Setting.plugin_redmine_telegram[:display_watchers] == 'yes'
 
-
-      Mailer.speak(msg, channel, attachment, token)      
```

Comment:
A little idea - send message to Project Chat, and after that - send messages for users.
It will be nice use-case